backend/src/controllers/user.controller.ts

```diff
@@ -41,9 +41,17 @@ export const getOne = async (req: Request, res: Response): Promise<Response> =>
 };
 
 export const create = async (req: Request, res: Response): Promise<Response> => {
-  const user = req.body;
+  const body = req.body as IUser;
   try {
-    const result = await User.create(user);
+    if (body.email) {
```

Comment:
How this is working when we have `unique` flag in model for `email` field? Is this validation required?

Reply:
If we have a unique field and we are trying to put an email that already exists it will throw an exception. I think the validation is required because the section doesn't have an email.